### PR TITLE
return output from triggerSmartblock if no target

### PR DESCRIPTION
**Pull request by mdroidian**

### CODE REVIEW

**Comment by mdroidian** [comment]
another option might be to edit `triggerSmarblock` to return the output via `processBlockUid` if `targetName` and `targetUid` are blank 

(instead of throwing an `Error)

**Comment by mdroidian** [comment]
Might need to be re-thought ... things like `<%FORM%>` and `<%OPENPAGE%>` will still run ... 🤔

File: src/index.ts [hunk 1]
```diff
@@ -320,8 +321,8 @@ export default runExtension(async ({ extensionAPI }) => {
         if (targetName) {
           throw new Error(`Could not find page with name ${targetName}`);
         } else {
-          throw new Error(
-            "Either the `targetName` or `targetUid` input is required"
+          return new Promise((resolve) =>
+            processBlockUid(srcUid).then(resolve)
```
Comment (dvargas92495) [inline src/index.ts:325]: We want:
- `resetContext({ targetUid: null, variables, triggerUid });`
- `const childNodes = getFullTreeByParentUid(srcUid).children`
- `processChildren({ nodes: childNodes })`

File: src/index.ts [hunk 2]
```diff
@@ -320,9 +323,13 @@ export default runExtension(async ({ extensionAPI }) => {
         if (targetName) {
           throw new Error(`Could not find page with name ${targetName}`);
         } else {
-          throw new Error(
-            "Either the `targetName` or `targetUid` input is required"
-          );
+          resetContext({
+            targetUid: undefined,
+            variables,
+            triggerUid: undefined,
```
Comment (mdroidian) [inline src/index.ts:329]: @dvargas92495 maybe just double check using `undefined` here.

Comment (mdroidian) [reply src/index.ts:329]: Will go ahead and merge, but let me know if something is off.

**Pull request merged**